package.json

```diff
@@ -138,8 +138,9 @@
     "fs-extra": "8.1.0",
     "https-browserify": "1.0.0",
     "identity-obj-proxy": "3.0.0",
-    "jest": "27.5.1",
+    "jest": "28.1.3",
     "jest-junit": "13.0.0",
+    "jsdom": "26.0.0",
```

Comment:
@brave/js-deps-reviewers we previously had an implicit dependency on `jsdom` but that package (presumably Jest) must have moved to an optionalPeerDependency